### PR TITLE
[error tracking] Custom fingerprint docs

### PR DESCRIPTION
**Pull request by GianlucaBortoli**
### What does this PR do?

Add public docs for the "custom fingerprint" new functionality in Error Tracking (see [RFC](https://docs.google.com/document/d/16y1qpl7w5G8X0uac0K9y2N6kdH_IWxU0ylMVU4BP4I8/edit#heading=h.3422lusznkmt)).

### Motivation

Need docs out before releasing the feature to customers.

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

**Comment by AyushK1** [comment]
cc: @alai97 please review

**Review by alai97 — APPROVED** [review]
Looks good for docs!

File: config/_default/menus/menus.en.yaml [hunk 1]
```diff
@@ -2457,6 +2462,10 @@ main:
     url: logs/error_tracking/backend
     parent: log_management_error_tracking
     weight: 603
+  - name: Custom Grouping
+    url: logs/error_tracking/custom_grouping
+    parent: log_management_error_tracking
```
Comment (alai97) [inline config/_default/menus/menus.en.yaml:2467]:
```suggestion
    parent: log_management_error_tracking
    identifier: log_management_error_tracking_custom_grouping
```

File: content/en/real_user_monitoring/error_tracking/custom_grouping.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+If you're already sending RUM events, add a new `error.fingerprint` attribute to your RUM error event.
+
+Here's an example for [collecting browser errors][2]:
+
+```javascript
+import { datadogRum } from '@datadog/browser-rum';
+
+// Send a custom error with context
+const err = new Error('Something went wrong');
```
Comment (milanboers) [inline content/en/real_user_monitoring/error_tracking/custom_grouping.md:48]:
```suggestion
const error = new Error('Something went wrong');
```
(doesn't match with the next line)

**Review by milanboers — APPROVED** [review]

**Comment by GianlucaBortoli** [comment]
@alai97 we're currently rolling out the feature. I'll ping you here to let you know when it's safe to merge the PR since I don't have enough permissions to do so.
Thanks a lot!

**Pull request merged**